lib/SILOptimizer/Transforms/SimplifyCFG.cpp

```diff
@@ -3190,6 +3190,7 @@ bool simplifyToSelectValue(SILBasicBlock *MergeBlock, unsigned ArgNum,
   }
   
   SmallVector<std::pair<SILValue, SILValue>, 8> Cases;
+  llvm::SmallSetVector<SILValue, 8> CaseLiterals;
```

Comment:
You don't need a SetVector because you don't iterate over the values. A plain SmallSet is sufficient